src/main/java/domain/attendance/Holiday.java

```diff
@@ -0,0 +1,42 @@
+package domain.attendance;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+public enum Holiday {
+    CHRISTMAS(12, 25),
+    NEW_YEAR(1, 1),
+    ;
+
+    private final int month;
+    private final int day;
+
+    Holiday(int month, int day) {
+        this.month = month;
+        this.day = day;
+    }
+
+    public static boolean has(LocalDateTime dateTime) {
+        int month = dateTime.getMonthValue();
+        int day = dateTime.getDayOfMonth();
+
+        for (Holiday value : values()) {
+            if (value.month == month && value.day == day) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public static boolean has(LocalDate date) {
```

Comment:
LocalDate 를 받는 것과 LocalDateTime  두 타입을 인자로 받는 동일한 역할의 메서드를 구현하신 배경이 궁금해요!

필요하다면 아래와 같이 구현하는 방법도 있을 것 같아요~
현재는 메서드 내용이 변경되면 동일한 곳을 한 번더 수정해야하니까요 😢 
```suggestion
    public static boolean has(LocalDateTime dateTime) {
        return has(dateTime.toLocalDate());
    }

    public static boolean has(LocalDate date) {
```

Reply:
원래 `boolean has(LocalDateTime dateTime);` 메서드만 있었는데, 개발을 하다보니 LocalDate만 있을 때에도 `has()` 메서드를 사용해야 했어요. 시간에 쫓겨 정신없이 개발하다보니 오버로딩만 생각나서 급하게 같은 메서드를 하나 더 추가했습니다😂

제안해주신 방법이 유지보수성에서 훨씬 좋을 것 같아요 좋은 방법 알려주셔서 감사합니다! 
추가로 생각해보니 `boolean has(LocalDate date);` 만 남겨두면 오버로딩이 필요없을 것 같아서 has() 메서드는 하나만 남겼습니다. 😄